packages/block-library/src/navigation/view.js

```diff
@@ -164,20 +167,20 @@ wpStore( {
 					}
 				},
 				handleMenuFocusout: ( store ) => {
-					const { context, selectors, event } = store;
+					const { context, event } = store;
 					// If focus is outside modal, and in the document, close menu
 					// event.target === The element losing focus
 					// event.relatedTarget === The element receiving focus (if any)
 					// When focusout is outsite the document,
 					// `window.document.activeElement` doesn't change.
 					if (
-						selectors.core.navigation.menuOpenBy( store ).click &&
```

Comment:
We shouldn't be careful about not calling `closeMenu` multiple times even if the menu is closed. If anything, we should make the side effects that happen when the menu is closed (inside the `closeMenu` function) more declarative so they only trigger when the status changes.

Reply:
If I am not mistaken, I added that check because the `focusout` event was triggered in the children, so it is being called each time we change focus over the elements of the submenu.

Reply:
I think that works fine now with the new implementation, doesn't it?